package.json

```diff
@@ -4,15 +4,15 @@
   "description": "`BinderHub`",
   "devDependencies": {
     "bootstrap": "^3.4.0",
-    "clipboard": "^1.7.1",
-    "css-loader": "^0.28.7",
-    "event-source-polyfill": "^0.0.12",
-    "extract-text-webpack-plugin": "^3.0.2",
```

Comment:
"extract-text-webpack-plugin" is deprecated (https://github.com/webpack-contrib/extract-text-webpack-plugin)